test/cli/test_log_runtime_archive_main.rb

```diff
@@ -238,11 +242,14 @@ def rate_mbps_to_bps(rate_mbps)
                 # Call 'transfer' function instead of 'watch' to call transfer once
                 def call_transfer(source_dir, server_port: nil)
                     updated_server_params = @server_params
+                    implicit_ftps = updated_server_params[:implicit_ftps]
+                    updated_server_params.delete(:implicit_ftps)
                     updated_server_params[:port] = server_port if server_port
                     args = [
                         "transfer",
                         source_dir,
-                        *updated_server_params.values
+                        *updated_server_params.values,
+                        "--implicit_ftps", implicit_ftps
```

Comment:
would this work? I expect boolean CLI arguments to be added by the flag being there or not, does it parse explicitly called `--bla true` args correctly?

Reply:
I tested it in CLI and it worked as expected, but you're not the first one asking this, I am going to change to the standard way of boolean CLI arguments.